packages/preview/touying/0.1.0/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2024 OrangeX4 <orangex4@qq.com>

```

Comment:
Since large parts are derived from polylux, this should also specify `Copyright (c) 2023 Andreas Kröpelin` in addition to the individual attributions.

Reply:
Oh, thanks, I only added it to typst.toml and forgot to change the LICENSE. It has now been added.